config.go

```diff
@@ -108,6 +108,14 @@ type HTTPConfig struct {
 	//
 	// Added in v4.2.0.
 	BasicAuth string
+
+	// OIDC SHOULD be use with HTTPS/SSL (unsecure without)
+	// Requires CORS set to specific origins.
+	// Return unauthorized on http requests unless valid OIDC access
+	// tokens sent through the Authorization header for all requests.
```

Comment:
nit: I find this can read a bit better with some changes, something like:
```suggestion
	// OIDC is only secure if using HTTPS/SSL.
	// Requires CORS set to specific origins.
	//
	// If enabled, HTTP requests without a valid OIDC access token in the Authorization
	// header will be rejected with Unauthorized 401.
```
maybe?